static/src/stylesheets/module/content/_media.global.scss

```diff
@@ -70,9 +70,17 @@
         }
     }
 }
-// if it's audio and there's body text, we display an advert
-.content--has-body .content__main-column--audio {
-    min-height: 274px;
+
+.content--has-body {
+    // if it's audio and there's body text, we display an advert
+    .content__main-column--audio {
+        min-height: 274px;
+    }
+
+    // increase height if we have sponsorship logos
+    &.paid-content--advertisement-feature .content__main-column--audio {
```

Comment:
I had tried that; the compiler didn't seem to like using the parent selector with a modifier.